### DISCUSSION TITLE
New powerpc big-endian ISO split

### DISCUSSION THREAD

**Discussion by kth5** [kth5/archpower#79] [Announcements]
Previously the `powerpc` ISO would include both 32bit and 64bit kernels to re-use the same ISO download/CD-R for both scenarios, such as running a 32bit userland on a PowerMac G5.

This has had the negative side effect to not let the ISO fit on a CD-R anymore (>800Mbyte).

With the kernels being a major impact on ISO sizes, it was a natural decision to split the ISOs up into two.

|ISO|Description|
|---------|-------------|
|https://archlinuxpower.org/iso/archpower-current-powerpc.iso|32bit big-endian `powerpc`|
|https://archlinuxpower.org/iso/archpower-current-powerpc6432.iso|32bit big-endian `powerpc` userland with 64bit `powerpc64` kernel|
|https://archlinuxpower.org/iso/archpower-current-powerpc64.iso|64bit big-endian `powerpc` userland with 64bit `powerpc64` kernel|

Please be aware of this since you must use a kernel matching your system as a 64bit kernel won't run in 32bit machines and vice versa. If you need address space for processes asking for more than 4GiB RAM, chose a pure `powerpc64` installer. If you are out for speed and efficiency, run `powerpc6432` on a 64bit system.

**Comment by Oichkatzelesfrettschen** [comment]
Good call! Thanks for keeping this CD-sized!
